pkg/image/resolve.go

```diff
@@ -97,29 +147,31 @@ func pickImagesFromValuesYAML(imagesSet map[string]map[string]bool, charts map[s
 	if err != nil {
 		return err
 	}
-	dataInterface := map[interface{}]interface{}{}
-	if err := yaml.Unmarshal(data, &dataInterface); err != nil {
+	chartValues := map[interface{}]interface{}{}
+	if err := yaml.Unmarshal(data, &chartValues); err != nil {
 		return err
 	}
 
-	walkthroughMap(dataInterface, func(inputMap map[interface{}]interface{}) {
-		generateImages(chartNameAndVersion, inputMap, imagesSet, osType)
+	var imagesFound []string
+	walkthroughMap(chartValues, func(inputMap map[interface{}]interface{}) {
+		generateImages(chartNameAndVersion, inputMap, imagesSet, &imagesFound, osType)
 	})
+	log.Printf("Images found in %s: %+v", path, imagesFound)
```

Comment:
I think this is a good replacement for the log that is being removed. A slice value should just be returned from the function here, pass it as a pointer is unnecessary and not idiomatic in this scenario: https://blog.golang.org/slices. I also think instead of passing this info back you could just print this where it is being appended.

Reply:
I'll update it to return a list rather than print each individual image. The reason for this is that I think printing a list of images (including empty for no images found) and the path to the chart it was found (including subcharts) is more useful compared to printing each individual image that was generated since then it would be harder to distinguish which images are from the chart and subchart.